src/credentials/DeviceAttestationVerifier.h

```diff
@@ -244,10 +244,14 @@ class DeviceAttestationVerifier
 
     // TODO: Validate Firmware Information
 
+    virtual CHIP_ERROR ValidateNodeOperationalCSRInformation(const ByteSpan & nocsrElementsBuffer,
```

Comment:
This needs to check that the nonce that was sent in the CSR request matches the nonce that came back in nocsrElementsBuffer, just as VerifyAttestationInformation does with the nonce sent in the attestation request.  Else there's no replay protection.

Also suggest this naming for consistency:

```suggestion
    virtual CHIP_ERROR VerifyNodeOperationalCSRInformation(const ByteSpan & nocsrElementsBuffer,
```

Finally, please add doxy headers.

Reply:
Addressed in latest commit